<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>goods_card</class>
 <widget class="QWidget" name="goods_card">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Карточка товара</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="add_barcode">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Добавить ШК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="view_barcode">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_barcode">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Удалить ШК</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="cancel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="addEgaisGood">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Добавить товар ЕГАИС</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="egais_goods">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeEgais">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Удалить товар егаис</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Записать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Название</string>
       </property>
       <property name="buddy">
        <cstring>goods_edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="goods_edit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Пользовательский код</string>
           </property>
           <property name="buddy">
            <cstring>code_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="code_edit">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Цена</string>
             </property>
             <property name="buddy">
              <cstring>price_edit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Единицы измерения</string>
             </property>
             <property name="buddy">
              <cstring>units_edit</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="price_edit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="units_edit">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Группа печати</string>
             </property>
             <property name="buddy">
              <cstring>group_edit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Обьем</string>
             </property>
             <property name="buddy">
              <cstring>volume_edit</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="group_edit">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="volume_edit">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QCheckBox" name="excise">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Акцизный</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="alcohol">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Алкогольный</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="goodsGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string>Товарная Группа</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Нажмите кнопку добавить товар, &lt;/p&gt;&lt;p&gt;Осканируйте акцизную марку, &lt;/p&gt;&lt;p&gt;Привяжите товар егаис&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>ОК</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
